bindings/java/src/main/com/apple/foundationdb/Database.java

```diff
@@ -161,24 +161,147 @@ default Transaction createTransaction() {
 	 */
 	double getMainThreadBusyness();
 
+	/**
+	 * Runs {@link #purgeBlobGranules(Function)} on the default executor.
+	 *
+	 * @param beginKey start of the key range
+	 * @param endKey end of the key range
+	 * @param purgeVersion version to purge at
+	 * @param force if true delete all data, if not keep data >= purgeVersion
+	 *
+	 * @return the key to watch for purge complete
```

Comment:
I must have missed this when we talked this morning.  How does this watch key work?